test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=std -source-filename=x -enable-cxx-interop -Xcc -stdlib=libc++ -tools-directory=%llvm_obj_root/bin -module-cache-path %t | %FileCheck %s  -check-prefix=CHECK-STD
```

Comment:
Hmm so it looks like here you're specifically testing libc++. I think it would be good to test with whatever stdlib the client's is using. 

I'm not sure libstdc++ will work until #38675 lands, so this is OK for now.